src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTab: Tab;
+  onSelectTab: (tab: Tab) => void
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTab,
+  onSelectTab,
+}) => {
+  return (
+    <div className="section">
+      <h1 className="title">
+        {`Selected tab is ${selectedTab.title}`}
+      </h1>
+
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => (
+              <li
+                key={tab.id}
+                data-cy="Tab"
+                className={classNames({
+                  'is-active': selectedTab === tab,
+                })}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => (
+                    tab.id !== selectedTab.id && onSelectTab(tab)
+                  )}
```

Comment:
```suggestion
                  onClick={() => handleTabClick(tab)}
```
Move handler logic to separate function.